packages/venia-concept/src/components/Icon/icon.js

```diff
@@ -13,11 +15,16 @@ class Icon extends Component {
     };
 
     render() {
-        const { attrs, classes, name } = this.props;
-        const svg = Feather.icons[name].toSvg(attrs);
-        const fn = () => ({ __html: svg });
+        const { attrs, classes, src: IconComponent } = this.props;
 
-        return <span className={classes.root} dangerouslySetInnerHTML={fn()} />;
+        return (
+            <span className={classes.root}>
+                <IconComponent
+                    size={attrs && attrs.width}
```

Comment:
@sirugh That would be nice, thanks! Let's make it a union type so that either a `{ height, width }` or an int works.

Reply:
I don't think `react-feather` takes height and width unfortunately. For example, [this icon](https://github.com/carmelopullara/react-feather/blob/master/src/icons/align-justify.js#L5-L18) takes `color, `size`, and `otherProps` which are spread. Though now that I look at it I wonder if that would spread/overwrite width if provided.

Reply:
My bad, I misunderstood. I guess what I mean is something like:

```suggestion
                    size={this.props.size && Number(this.props.size) || attrs && Number(attrs.width)}
```

which would allow:
```jsx
<Icon src={Foo} size={16} />
<Icon src={Foo} size="16" />
<Icon src={Foo} attrs={{ width: 16 }} />
<Icon src={Foo} attrs={{ width: "16" }} />
```